app/database/operator/server_data_operator/handlers/group.ts

```diff
@@ -106,6 +109,66 @@ const GroupHandler = (superclass: any) => class extends superclass implements Gr
 
         return records;
     };
+
+    /**
+     * handleGroupTeamsForTeam: Handler responsible for the Create/Update operations occurring on the GroupTeam table from the 'Server' schema
+     * @param {HandleGroupTeamsForTeamArgs} groupTeamsArgs
+     * @param {string} groupTeamsArgs.teamId
+     * @param {GroupTeam[]} groupTeamsArgs.groupTeams
```

Comment:
See other comment about types below

Reply:
This is the incorrect one - I'm going to remove the additional comments as they aren't quite correct; we have the main `HandleGroupTeamsForTeamArgs` already, the others aren't necessary and we seem to be going back and forth on them.